docs/iris/src/further_topics/lenient_maths.rst

```diff
@@ -0,0 +1,281 @@
+However, this behaviour may be controlled via the thread-safe ``LENIENT["maths"]``
+runtime option,
+
+.. doctest:: lenient-example
+
+    >>> from iris.common import LENIENT
+    >>> print(LENIENT)
+    Lenient(maths=True)
+
+Which may be set and applied globally thereafter for Iris within the current
+thread of execution,
+
+.. doctest:: lenient-example
+
+    >>> LENIENT["maths"] = False  # doctest: +SKIP
+    >>> print(LENIENT)  # doctest: +SKIP
+    Lenient(maths=False)
+
+Or alternatively, temporarily alter the behaviour of cube maths only within the
+scope of the ``LENIENT`` `context manager`_,
+
+.. doctest:: lenient-example
+
+    >>> print(LENIENT)
+    Lenient(maths=True)
+    >>> with LENIENT.context(maths=False):
+    ...     print(LENIENT)
+    ...
+    Lenient(maths=False)
+    >>> print(LENIENT)
+    Lenient(maths=True)
+
+
+Strict example
+==============
+
+Now that we know how to control the underlying behaviour of cube maths,
+let's return to our :ref:`lenient example <lenient example>`, but this
+time perform **strict** cube maths instead,
+
+.. doctest:: lenient-example
+
+    >>> with LENIENT.context(maths=False):
+    ...     difference = experiment - control
+    ...
+    >>> print(difference)
+    unknown / (K)                       (model_level_number: 15; grid_latitude: 100; grid_longitude: 100)
+         Dimension coordinates:
+              model_level_number                           x                  -                    -
+              grid_latitude                                -                  x                    -
+              grid_longitude                               -                  -                    x
+         Auxiliary coordinates:
+              atmosphere_hybrid_height_coordinate          x                  -                    -
+              sigma                                        x                  -                    -
+              surface_altitude                             -                  x                    x
+         Derived coordinates:
+              altitude                                     x                  x                    x
+         Scalar coordinates:
+              time: 2009-09-09 17:10:00
+         Attributes:
+              source: Data from Met Office Unified Model 7.04
+
+Although the numerical result of this strict cube maths operation is identical,
+it is not as rich in metadata as the :ref:`lenient alternative <lenient example>`.
+In particular, it does not contain the ``forecast_period`` and ``forecast_reference_time``
+**scalar coordinates**, or the ``experiment-id`` in the ``attributes`` dictionary.
+
+This is because strict cube maths, in general, will only return common metadata
+and common coordinates that are :ref:`strictly equivalent <strict equality>`.
+
+
+Finer detail
+============
+
+In general, if you want preserve as much metadata and coordinate information as
```

Comment:
if you want **to** preserve...

Reply:
@rcomer Awesome spot :eyes: ... that section is hot off the press, so thanks for taking a look at it, much appreciated :+1: